docs/guides/Mac Deployment.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+LiveCode 8.0 DP15 introduced the ability to build 64-bit standalones for MacOSX after a successful sponsorship campaign on the LiveCode Feature Exchange.
```

Comment:
Let's just say "LiveCode 8" here.  Presumably, if people have got the ability to deploy to Mac OS 64-bit, then they must have a sufficiently recent DP!